### ISSUE TITLE
Missing SD-Card in comparison table

### ISSUE THREAD

**Issue by jumpjack** [Xinyuan-LilyGO/TTGO_TWatch_Library#196]
This very important feature is missing in the comparison table. The ability to have a "wifi pen drive" strictly tied to your wrist, so you can't drop/loose/burn it, is very interesting, I think it should be advertised more. 
Why isn't the SD card present in next versions?
Which is the maximum size allowed?

**Comment by zhangqinghuan** [comment]
Yes, I endorse your idea, so I had to opt for the v2 version (my English may not be very good)
